docker/zap_common.py

```diff
@@ -423,3 +509,12 @@ def write_report(file_path, report):
             report = report.encode('utf-8')
 
         f.write(report)
+
+@hook(wrap=True)
+def zap_import_context(zap, context_file):
+    # handle the context file, cant use base_dir as it might not have been set up
```

Comment:
This should be removed, does not apply here.

Reply:
Why do you say that?

Reply:
There's no `base_dir` here, the comment was from callers' scope.

Reply:
Ah gotcha yes!